src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js

```diff
@@ -0,0 +1,310 @@
+import LostItemsList, {
+  triggerOnSort,
+  basicListFormatter,
+  COLUMNS_NAME,
+  visibleColumns,
+  columnMapping,
+  columnWidths,
+} from './LostItemsList';
+import {
+  ActualCostModal,
+  ActualCostConfirmModal,
+  InstanceDetails,
+} from './components';
+import {
+  ACTUAL_COST_CONFIRM_MODAL_DEFAULT,
+  ACTUAL_COST_DEFAULT,
+  ACTUAL_COST_MODAL_DEFAULT,
+  PAGE_AMOUNT,
+} from '../../../constants';
+
+const patronName = 'name';
+
+jest.mock('./components', () => ({
+  ...jest.requireActual('./components'),
+  ActualCostModal: jest.fn(() => null),
+  ActualCostConfirmModal: jest.fn(() => null),
+  InstanceDetails: jest.fn(() => null),
+}));
+jest.mock('./util', () => ({
+  getPatronName: jest.fn(() => patronName)
+}));
+jest.mock('react-intl', () => ({
+  ...jest.requireActual('react-intl'),
+  FormattedMessage: jest.fn(({ id }) => id),
+  FormattedTime: jest.fn(({ value }) => value),
+  FormattedDate: jest.fn(({ value }) => value),
+}));
+
+const initialProps = {
+  contentData: [],
+  totalCount: 10,
+  onNeedMoreData: jest.fn(),
+  emptyMessage: 'empty message',
+  onSort: jest.fn(),
+  sortOrder: '-',
+};
+const testIds = {
+  lostItemsList: 'lostItemsList',
+};
+const messageIds = {
+  lossType: 'ui-users.lostItems.list.filters.lossType.agedToLost',
+};
+
+describe('LostItemsList', () => {
+  describe('triggerOnSort', () => {
+    const event = {};
+    const onSort = jest.fn();
+
+    describe('when "meta.name" is equal "COLUMNS_NAME.ACTION"', () => {
+      it('should return "noop"', () => {
+        const meta = {
+          name: COLUMNS_NAME.ACTION,
+        };
+
+        expect(triggerOnSort(event, meta, onSort)).toEqual(noop);
+      });
+    });
+
+    describe('when "meta.name" is not equal "COLUMNS_NAME.ACTION"', () => {
+      it('should "onSort" with correct arguments', () => {
+        const meta = {
+          name: 'test',
+        };
+
+        triggerOnSort(event, meta, onSort);
+
+        expect(onSort).toHaveBeenCalledWith(event, meta);
+      });
+    });
+  });
+
+  describe('basicListFormatter', () => {
+    const basicActualCostRecord = {
+      user: {
+        patronGroup: 'patronGroup',
+      },
+      item: {
+        permanentLocation: 'permanentLocation',
+      },
+      feeFine: {
+        owner: 'owner',
+        type: 'type',
+      },
+      lossType: 'Aged to lost',
+      lossDate: 'lossDate',
+    };
+
+    describe('Patron formatter', () => {
+      describe('when patron group is presented', () => {
+        beforeEach(() => {
+          render(basicListFormatter[COLUMNS_NAME.PATRON](basicActualCostRecord));
+        });
+
+        it('should render "patronName"', () => {
+          expect(screen.getByText(patronName)).toBeVisible();
+        });
+
+        it('should render "patronGroup"', () => {
+          expect(screen.getByText(`(${basicActualCostRecord.user.patronGroup})`)).toBeVisible();
+        });
+      });
+
+      describe('when patron group is not presented', () => {
```

Comment:
patronGroup this is required field for user creation. Could you please check necessity of this tests?

Reply:
Removed